sherpa/astro/instrument.py

```diff
@@ -18,23 +18,17 @@
 #  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
 
-"""
-Models of common Astronomical models, particularly in X-rays.
+"""Models of common Astronomical models, particularly in X-rays.
 
 The models in this module include support for instrument models that
 describe how X-ray photons are converted to measurable properties,
 such as Pulse-Height Amplitudes (PHA) or Pulse-Invariant channels.
-These 'responses' are assumed to follow OGIP standards, such as
-[OGIP92-002]_.
-
-References
-----------
-
-.. [OGIP92-002] OGIP Calibration Memo CAL/GEN/92-002, "The Calibration Requirements
-       for Spectral Analysis (Definition of RMF and ARF file formats)",
-       Ian M. George1, Keith A. Arnaud, Bill Pence, Laddawan Ruamsuwan and
-       Michael F. Corcoran,
-       https://heasarc.gsfc.nasa.gov/docs/heasarc/caldb/docs/memos/cal_gen_92_002/cal_gen_92_002.html
+These 'responses' are assumed to follow OGIP standards, such as `OGIP
+Calibration Memo CAL/GEN/92-002, "The Calibration Requirements for
+Spectral Analysis (Definition of RMF and ARF file formats)", Ian
+M. George1, Keith A. Arnaud, Bill Pence, Laddawan Ruamsuwan and
```

Comment:
```suggestion
M. George, Keith A. Arnaud, Bill Pence, Laddawan Ruamsuwan and
```

not sure where that came from...